src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.ts

```diff
@@ -109,7 +110,10 @@ export class ServiceFormComponent extends CdForm implements OnInit {
                 return !this.RGW_SVC_ID_PATTERN.test(value);
               })
             ]
-          )
+          ),
+          CdValidators.custom('uniqueName', (service_id: string) => {
+            return this.serviceIds && this.serviceIds.indexOf(service_id) !== -1;
```

Comment:
[JS ES6 introduced the `.includes()` method](https://www.w3schools.com/js/js_es6.asp), which is nicer to read than the old `indexOf()`:
```suggestion
            return this.serviceIds && !this.serviceIds.includes(service_id);
```

Reply:
Thanks @epuertat . Updated the PR